jupyterlab_comments/jupyterlab_comments/git_commands.py

```diff
@@ -78,6 +92,10 @@ def get_code_review_comments(self, file_path_from_repo_root, git_root_dir):
 		"""
 		review_string = self.run(git_root_dir, 'appraise', 'show',
 									'-json')
+		if review_string is None:
+			print('No open code reviews')
+			return None
+
 		review_json = json.loads(review_string)
 		if not review_json:
```

Comment:
not sure if we will hit this section if review_string is already None above. Line 95.

Reply:
with git appraise, I believe review_string will be None if there are no open code reviews, but it will be an empty string if there is an open code review but with no comments on that code review yet. I added the first check for None so I can safely call json.loads(review_string)

Reply:
Reviews have more than just the comments, so if you want to explicitly check if there are no comments (or no matching comments), then we need to check that the comment threads field is missing or empty.

Reply:
Although, taking another look, this is really just a safeguard, and I think it is fine for it to be overly cautious by checking for a situation that might not happen.

The checks for no comments is done well in the subsequent code and strictly speaking the `review_json` field could still be falsey even if the string read in was not `None`.